### PR TITLE
Add network related state

### PR DESCRIPTION
**Pull request by taulinger**
- adds a network-related slice of state
- intercepts fetch calls to update networks fetching property
- animates BA logo while fetching
- remove double 'it' expressions in tests

### CODE REVIEW

**Comment by taulinger** [comment]
Build shows overall decreased coverage due to poor coverage score of `Autocomplete.js` (https://coveralls.io/jobs/78819405)
Must be an error, did not change anything there. Normally `Autocomplete.js`  has around 97%.

File: src/services/HttpService.js [hunk 1]
```diff
@@ -57,7 +64,7 @@ export class HttpService {
 	 */
 	async post(resource, data, contentType, options = {}) {
 		const fetchOptions = {
-			mode: 'cors',
+			mode: HttpService.DefaultRequestMode,
 			method: 'POST',
```
Comment (thiloSchlemmer) [inline src/services/HttpService.js:68]: To be consequent, the request method-types could also be extracted to private CONST

**Review by thiloSchlemmer — APPROVED** [review]
Good job.... only tiny remark found

File: src/services/HttpService.js (hunk 1 above)
Comment (taulinger) [reply src/services/HttpService.js:68]: Need more explanation...

Comment (taulinger) [reply src/services/HttpService.js:68]: Probably I should name the static getter `DEFAULT_REQUEST_MODE` to indicate it has public access?

**Pull request merged**